docs: add new 2.1 English docs and revise Chinese docs (#196)
chaos-mesh/website: docs/simulate-physical-machine-chaos.md

## Introduction to PhysicalMachineChaos

You can use PhysicalMachineChaos to simulate the faults of network, disk, pressure, JVM, time, and others in physical or virtual machines.

## Deploy Chaosd server

Before creating PhysicalMachineChaos experiments using Chaos Mesh, you need to deploy Chaosd in service mode to all physical or virtual machines that are going to be injected with faults. For the deployment method of Chaosd, refer to [Download and deploy Chaosd](chaosd-overview.md#download-and-deploy).

After the deployment is complete, run the following command to start Chaosd in service mode:

```bash
chaosd server --port 31767
```

:::note

When using Chaos Mesh v2.1.0, you need to deploy Chaosd [v1.1.0](https://github.com/chaos-mesh/chaosd/releases/tag/v1.1.0).

:::

## Create experiments using Chaos Dashboard

1. Open Chaos Dashboard, and click **NEW EXPERIMENT** on the page to create a new experiment:

   ![create a new experiment](./img/create-new-exp.png)

2. Click **PHYSICAL** in **EXPERIMENTS TYPE** and choose a specific type of the experiment, such as **NETWORK ATTACK**. Then, choose a specific behavior and fill out the corresponding configurations:

>    ![PhysicalMachineChaos experiment](./img/physicalmachinechaos-exp.png)

Reviewer: 没有图片？

3. Fill out the experiment information to specify the range and the planned duration of the experiment:

>    ![PhysicalMachineChaos experiment information](./img/physicalmachinechaos-exp-info.png)

Reviewer: 图片没有上传吗

4. Submit the experiment.

## Create experiments using an YAML file

> 1. Write experimental configurations to the "physicalmachine.yaml" file. For example:

Reviewer: ```suggestion
1. Write the experiment configurations to the "physicalmachine.yaml" file. For example:
```

   ```yaml
   apiVersion: chaos-mesh.org/v1alpha1
   kind: PhysicalMachineChaos
   metadata:
     name: physical-network-delay
     namespace: chaos-testing
   spec:
     action: network-delay
     address:
       - 172.16.112.130:31767
     network-delay:
       device: ens33
       ip-address: 140.82.112.3
       latency: 1000ms
     duration: '10m'
   ```

>    These experimental configurations send HTTP requests to Chaosd service in the specified physical machines or virtual machines to trigger network latency experiments.

Reviewer: ```suggestion
   The experiment configurations above send HTTP requests to Chaosd service in the specified physical machines or virtual machines to trigger network latency experiments.
```

> 2. Create experiments using `kubectl`. The commands are as follows:

Reviewer: ```suggestion
2. Create the experiment using `kubectl`. The command is as follows:
```

   ```bash
   kubectl apply -f physicalmachine.yaml
   ```

### Configuration description

| Configuration item | Type | Description	 | Default value | Required | Example |
| :-- | :-- | :-- | :-- | :-- | :-- |

> | action | string | Defines the actions of physical machines faults, optional values are as follows: "stress-cpu", "stress-mem", "disk-read-payload", "disk-write-payload", "disk-fill", "network-corrupt", "network-duplicate", "network-loss", "network-delay", "network-partition", "network-dns", "process", "jvm-exception", "jvm-gc", "jvm-latency", "jvm-return", "jvm-stress", "jvm-rule-data", "clock" | No | Yes | "stress-cpu" |
> | address | string array | Selects the address of Chaosd service to inject faults | [] | Yes | ["192.168.0.10:31767"] |
> | duration | string | Specifies the duration of experiments | No | Yes | 30s |

Reviewer: ```suggestion
| action | string | Defines the actions of physical machines faults, optional values are as follows: "stress-cpu", "stress-mem", "disk-read-payload", "disk-write-payload", "disk-fill", "network-corrupt", "network-duplicate", "network-loss", "network-delay", "network-partition", "network-dns", "process", "jvm-exception", "jvm-gc", "jvm-latency", "jvm-return", "jvm-stress", "jvm-rule-data", "clock" | None | Yes | "stress-cpu" |
| address | string array | Selects the address of Chaosd service to inject faults | [] | Yes | ["192.168.0.10:31767"] |
| duration | string | Specifies the duration of experiments | None | Yes | 30s |
```
`No` 可能有歧义，让用户以为默认值是 `"No"`.

> Each fault action has own specific configurations. The following section introduces various fault types and their corresponding configuration methods.

Reviewer: ```suggestion
Each fault action has its own specific configurations. The following section introduces various fault types and their corresponding configuration methods.
```

#### CPU stress

To simulate a CPU stress scenario, you need to set the action to "stress-cpu". For the corresponding configurations, refer to [Parameters of simulating CPU stress](simulate-heavy-stress-in-physical-nodes.md#parameters-of-simulating-cpu-stress).

#### Memory stress

To simulate a memory stress scenario, you need to set the action to "stress-mem". For the corresponding configurations, refer to [Parameters of simulating memory stress](simulate-heavy-stress-in-physical-nodes.md#parameters-of-simulating-memory-stress).

#### Disk read load

To simulate a disk read load scenario, you need to set the action to "disk-read-payload". For the corresponding configurations, refer to [Parameters of  simulating disk read load](simulate-disk-pressure-in-physical-nodes.md#parameters-of-simulating-disk-read-load).

#### Disk write load

> To simulate a disk write load scenario, you need to set the action to "disk-write-payload". For the corresponding configurations, refer to [Parameters of  simulating disk write load](simulate-disk-pressure-in-physical-nodes.md#parameters-of-simulating-disk-write-load).

Reviewer 2: ```suggestion
To simulate a disk write load scenario, you need to set the action to "disk-write-payload". For the corresponding configurations, refer to [Parameters of simulating disk write load](simulate-disk-pressure-in-physical-nodes.md#parameters-of-simulating-disk-write-load).
```
